# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1367210856 14400
#      Mon Apr 29 00:47:36 2013 -0400
# Node ID fe6beca1581391354d9ce8e75aff16696a32a816
# Parent  4d2b0104ee05ab2f9f7a849a29b7ff5edcc396bc
Remove resample from list of missing functions

* __unimplemented__.m: Remove resample from list, it is not a core
function.

diff --git a/scripts/help/__unimplemented__.m b/scripts/help/__unimplemented__.m
--- a/scripts/help/__unimplemented__.m
+++ b/scripts/help/__unimplemented__.m
@@ -327,17 +327,16 @@ function list = missing_functions ()
   "propedit",
   "propertyeditor",
   "publish",
   "qmr",
   "quad2d",
   "rbbox",
   "reducepatch",
   "reducevolume",
-  "resample",
   "root",
   "rotate",
   "rotate3d",
   "selectmoveresize",
   "sendmail",
   "serial",
   "setpixelposition",
   "showplottool",
